src/yunohost/tests/test_regenconf.py

```diff
@@ -0,0 +1,80 @@
+import glob
```

Comment:
MOAR TESTS \o/

Please could you add:

```yml
test-regenconf:
  extends: .tests
  stage: tests
  script:
    - cd src/yunohost
    - py.test tests/test_regenconf.py
```

At the end of this file?

Reply:
I can explain a little if you want:

```yml
test-regenconf:
```
Is the name, you can see the names of all the jobs here: https://gitlab.com/yunohost/yunohost/pipelines/137564286

```yml
  extends: .tests
```
Here we use the parameters from the `.tests` section. https://github.com/YunoHost/yunohost/blob/b392efdf85f3f9528772ff76dd40ad3239613bef/.gitlab-ci.yml#L5-L26

```yml
  stage: tests
```
Uuh, the stage (we have two stage for now, Postinstall and Tests) https://gitlab.com/yunohost/yunohost/pipelines/137564286

```yml
  script:
    - cd src/yunohost
    - py.test tests/test_regenconf.py
```

And finally the bash script to run during the test of this section.

Reply:
![alrightythen](https://user-images.githubusercontent.com/4533074/80043296-95a06600-8501-11ea-800f-d1357737c08f.gif)

Done in https://github.com/YunoHost/yunohost/pull/946/commits/817e45108631f1979e1443584713ea36a7d26643 , we'll see if CI is happy ;)